app/about-new/constants/whatWeOfferSection.ts

```diff
@@ -0,0 +1,38 @@
+type whatWeOfferItemType = {
+  header: string;
+  textContent: string;
+  href: string;
+};
+
+export const WHAT_WE_OFFER_CONTENT: whatWeOfferItemType[] = [
```

Comment:
types should be capitalized
```suggestion
export const WHAT_WE_OFFER_CONTENT: WhatWeOfferItemType[] = [
```